up42/_version.txt

```diff
@@ -1 +1 @@
-0.11.0
\ No newline at end of file
+0.11.1
```

Comment:
This is more than a bug fix, therefore need to bump to 0.12.0
```suggestion
0.12.0
```

Reply:
thanks